### PR TITLE
feat(icon): add branch, expert-node icons to library

### PR DESCRIPTION
**Pull request by shefaligupta**
## Description

[JIRA](https://dialpad.atlassian.net/browse/DP-56010)
Add branch, expert-node icons to the library. - Approved by @dialpad-marshall and Paolo.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

## Obligatory GIF (super important!)
1_qIkgSWsthjR8tSOF9X1TIw

### CODE REVIEW

File: docs/_data/svg-system.json [hunk 1]
```diff
@@ -155,6 +155,12 @@
     "vue": "IconBasketball",
     "desc": "Communicates basketball or, more generally, sports."
   },
+  {
+    "name": "Branch",
+    "file": "branch",
+    "vue": "IconBranch",
+    "desc": "Represents a branch in IVR workflows. Can be used elsewhere when needing to represent branch / branching / flows."
```
Comment (braddialpad) [inline docs/_data/svg-system.json]: This one is pretty re-usable and not necessarily specific to just IVR, can probably just say "Represents a branch"

File: docs/_data/svg-system.json [hunk 2]
```diff
@@ -449,6 +455,12 @@
     "vue": "IconError",
     "desc": "Communicates that something reached an unsuccessful state."
   },
+  {
+    "name": "Expert Node",
+    "file": "expert-node",
+    "vue": "IconExpertNode",
+    "desc": "Represents an expert node in IVR workflows. This is specific for now but we will be able to reuse this icon. May be used in other IVR experiences which is why the name sounds IVR specific since the word “expert” would be too vague in representation."
```
Comment (braddialpad) [inline docs/_data/svg-system.json]: Probably don't need all this explanation, it's a bit too much for the tooltip on the documentation site. Can just be "Represents an expert node in IVR workflows."

Comment (shefaligupta) [reply docs/_data/svg-system.json]: I thought so. :)

**Review by braddialpad — APPROVED** [review]
Great thanks!

**Pull request merged**